Resources/Textures/_Harmony/Objects/Weapons/Grenades/kudzugrenade.rsi/meta.json

```
@@ -0,0 +1,18 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "Taken from goonstation at https://github.com/goonstation/goonstation/pull/13630, recolor and additions made by @Flarbergaster (Github)",
```

Your attribution cites a pull request, it should cite a specific commit. Also, let's bring the attribution up to new standard (my own).

```suggestion
    "copyright": "Taken from goonstation at https://github.com/goonstation/goonstation/pull/13630. Modified by Flarbergaster (Github)",
```